<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>指定医生处方情况</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/back.jpg);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="0" colspan="3" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>指定医生处方情况</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>指定医生姓名：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>指定科室：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>医生编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>医生姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>患者姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>处方号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>药品</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>其他</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="4" column="1" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="background_image.qrc"/>
 </resources>
 <connections/>
</ui>
